Makefile

```diff
@@ -0,0 +1,21 @@
+API = ${PWD}/apps/api/src
+GET_FILES = ${PWD}/apps/get-files/src
+START_EXECUTION = ${PWD}/apps/start-execution/src
+UPLOAD_LOG = ${PWD}/apps/upload-log/src
+PROCESS_NEW_GRANULES = ${PWD}/apps/process-new-granules/src
+DYNAMO = ${PWD}/lib/dynamo
+export PYTHONPATH := ${API}:${GET_FILES}:${START_EXECUTION}:${UPLOAD_LOG}:${PROCESS_NEW_GRANULES}:${DYNAMO}
```

Comment:
I'm not sure we want the `:=` here -- if users already have a `PYTHONPATH` this won't do anything.

Might be better to do an `=` and blow away any user `PYTHONPATH` for these tests. 

This also requires you to be at the repository root, which I don't think is a problem